frontend/src/lib/components/DefinitionPopup/utils.ts

```diff
@@ -41,18 +41,18 @@ export function formatTimeFromNow(day?: string): string {
     return day ? dayjs.utc(day).fromNow() : '-'
 }
 
-export function getSingularType(type: TaxonomicFilterGroupType): string {
+export function getSingularType(type: TaxonomicFilterGroupType, shouldSimplifyActions: boolean = false): string {
```

Comment:
I've noticed you end up needing to propagate the flag into functions a lot which might have slightly different names. Worht adding the `TODO: remove when "simplify-actions" FF is released` into these lines too so it's easy to trace later on

Reply:
Probably add a comment at every single place where this happens so it's easy to remove after